__tests__/app.test.js

```diff
@@ -85,12 +85,66 @@ describe("CORE: GET - /api/articles/:article_id", () => {
         expect(body).toEqual({Error: "400, Invalid ID"})
       })
     })
-    test('400: valid ID that doesnt exist outputs useful error message', () => {
+    test('404: valid ID that doesnt exist outputs useful error message', () => {
       return request(app)
       .get("/api/articles/1000")
-      .expect(400)
+      .expect(404)
       .then(({body}) => {
         expect(body).toEqual({ Error: 'ID Does Not Exist' })
       })
     })
-  })
\ No newline at end of file
+  })
+
+  describe("CORE: GET - /api/articles", () => {
+    test("200: Endpoint should contain all article objects except body", () => {
+        return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.length).not.toBe(0)
+            body.forEach((article) => {
+              expect(article).toHaveProperty("article_id", expect.any(Number));
+              expect(article).toHaveProperty("title", expect.any(String)); 
+              expect(article).toHaveProperty("topic", expect.any(String)); 
+              expect(article).toHaveProperty("author", expect.any(String)); 
+              expect(article).toHaveProperty("created_at", expect.any(String)); 
+              expect(article).not.toHaveProperty("body"); 
+              expect(article).toHaveProperty("votes", expect.any(Number)); 
+              expect(article).toHaveProperty("article_img_url", expect.any(String)); 
+            });
+          });
+      });
+      test("200: Articles are correctly ordered (Descending by Date Created)", () => {
+        return request(app)
+          .get("/api/articles")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.length).not.toBe(0)
+             
+              const regex = /([TZ\W])/g
+  
+              const dateFormat1 = body[0].created_at.replace(regex, " ")
+              let arrayDate = dateFormat1.split(" ")
+
+              body.forEach((article) => {
+                const currentProperty = article.created_at
+                const dateFormat2 = currentProperty.replace(regex, " ")
+                const currentDate = dateFormat2.split(" ")
+
+                index = 0
+                  currentDate.forEach((num) => {
+                    const formatNum =  Number(num)
+                
+                    expect(formatNum).toBeLessThanOrEqual(Number(arrayDate[index]))
+
+                  })
+                arrayDate = currentDate 
+            })
+          });
+      });
+      test('404: Incorrect url input outputs a useful error', () => {
+        return request(app)
+        .get("/api/fakepath")
+        .expect(404)
+      })
```

Comment:
Unless you're testing for something more specific like an error message, this test as it stands is only testing that express is working as intended. Which is not for us to test.

If you wanted to test for a generic bad path it should be in it's own describe block.